.github/workflows/publish_to_NuGet.yml

```diff
@@ -22,3 +22,7 @@ jobs:
         env:
           NUGET_AUTH_TOKEN: ${{secrets.NUGET_AUTH_TOKEN}}
         run: dotnet nuget push Release/*.nupkg --skip-duplicate --no-symbols true --source https://api.nuget.org/v3/index.json -k ${NUGET_AUTH_TOKEN}
+      - name: Publish .snupkg to NuGet
```

Comment:
If we change directory to `Release` nuget push "might?" pick up the .snupkg package automatically. But maybe better to be specific and do two pushes